### PR TITLE
Update for 1.12.2

### PR DESCRIPTION
**Pull request by cosmicdan**
Also a couple other tiny improvements

### CODE REVIEW

File: src/main/java/me/guichaguri/tickratechanger/TickrateChanger.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 @TransformerExclusions({"me.guichaguri.tickratechanger"})
 public class TickrateChanger implements IFMLLoadingPlugin, IFMLCallHook {
 
-    public static TickrateChanger INSTANCE;
+    private static final TickrateChanger INSTANCE = new TickrateChanger();
```
Comment (Guichaguri) [inline src/main/java/me/guichaguri/tickratechanger/TickrateChanger.java]: Why do you initialize it here? Forge already creates an instance

File: build.gradle [hunk 2]
```diff
@@ -30,6 +30,17 @@ jar {
...
 
+// Create deobf dev jars
+task deobfJar(type: Jar) {
+    from sourceSets.main.output
+    classifier = 'deobf'
+	manifest {
+        attributes 'FMLCorePlugin': 'me.guichaguri.tickratechanger.TickrateChanger',
+                   'FMLCorePluginContainsFMLMod': 'true',
+                   'FMLAT': 'tickratechanger_at.cfg'
+    }
```
Comment (Guichaguri) [inline build.gradle]: Can you fix the indentation here?

**Review by Guichaguri — CHANGES_REQUESTED** [review]

**Comment by cosmicdan** [comment]
Done.

I changed TickrateChanger to an explicit singleton because I was getting an NPE crash on that INSTANCE field - but now I realize it was because I was missing that coremod arg on my run configuration :)

**Review by Guichaguri — APPROVED** [review]

**Comment by Guichaguri** [comment]
Thank you :)

**Pull request merged**